prov/verbs/src/verbs_domain.c

```diff
@@ -140,6 +149,26 @@ static int fi_ibv_mr_regattr(struct fid *fid, const struct fi_mr_attr *attr,
 			0, attr->requested_key, flags, mr, attr->context);
 }
 
+static int fi_ibv_domain_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
```

Comment:
As this is RDM specific code, please rename prefix like  fi_ibv_rdm_ where it's needed